packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -50,6 +52,54 @@ enum TextOverflow {
   visible,
 }
 
+/// The measurements of a character (or a sequence of visually connected
+/// characters) within a paragraph.
+///
+/// See also:
+///
+///  * [TextPainter.getClosestGlyphForOffset], which finds the [GlyphInfo] of
+///    the glyph(s) onscreen that's closest to the given [Offset].
+@immutable
+final class GlyphInfo {
```

Comment:
Had to copy the code from dart:ui since I forgot to expose the constructor. I'll add the constructor there and remove this class.